commit 752abfadcb8b60a1598024aea49f289fc36508c5
Author: Nickhu <nickhu@andestech.com>
Date:   Thu Oct 18 16:37:55 2018 +0800

    nds32: Fix gcc 8.0 compiler option incompatible.
    
    [ Upstream commit 4c3d6174e0e17599549f636ec48ddf78627a17fe ]
    
    When the kernel configs of ftrace and frame pointer options are
    choosed, the compiler option of kernel will incompatible.
            Error message:
                    nds32le-linux-gcc: error: -pg and -fomit-frame-pointer are incompatible
    
    Signed-off-by: Nickhu <nickhu@andestech.com>
    Signed-off-by: Zong Li <zong@andestech.com>
    Acked-by: Greentime Hu <greentime@andestech.com>
    Signed-off-by: Greentime Hu <greentime@andestech.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/nds32/mm/Makefile b/arch/nds32/mm/Makefile
index 6b6855852223..7c5c15ad854a 100644
--- a/arch/nds32/mm/Makefile
+++ b/arch/nds32/mm/Makefile
@@ -4,4 +4,8 @@ obj-y				:= extable.o tlb.o \
 
 obj-$(CONFIG_ALIGNMENT_TRAP)	+= alignment.o
 obj-$(CONFIG_HIGHMEM)           += highmem.o
-CFLAGS_proc-n13.o		+= -fomit-frame-pointer
+
+ifdef CONFIG_FUNCTION_TRACER
+CFLAGS_REMOVE_proc.o     = $(CC_FLAGS_FTRACE)
+endif
+CFLAGS_proc.o              += -fomit-frame-pointer
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1010_752abfadcb8b60a1598024aea49f289fc36508c5.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1010_752abfadcb8b60a1598024aea49f289fc36508c5.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
